keps/693-multikueue/README.md

```diff
@@ -0,0 +1,262 @@
+# KEP-693: MultiKueue
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+    - [Follow ups ideas](#follow-ups-ideas)
+  - [Test Plan](#test-plan)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+    - [E2E tests](#e2e-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+Introduce an new AdmissionCheck (called MultiKueue) with dedicated API
+and controller that will provide multi-cluster capabilities to Kueue.
+
+## Motivation
+Many of Kueue's users are running multiple clusters and would like to
+have a way to easily distribute batch jobs across them to keep all of
+them utilized. Without a global distribution point, some clusters may
+get less jobs they are able to process, while the others get more, leading
+to underutilization and higher costs. 
+
+### Goals
```

Comment:
I suggest that we add a point about warning in the documentation that MutliKueue (for batch Jobs) should only be used against kubernetes version that support the`managed-by` label. I think this is important to reduce the risk of users deploying MultiKueue on old versions, where two controllers are running, see here: https://github.com/kubernetes/enhancements/blob/be8ebccf4506055b1249ad798a9d902d676b7d4e/keps/sig-apps/4368-support-managed-by-label-for-batch-jobs/README.md#two-controllers-running-at-the-same-time-on-old-version

Alternatively this could go to the Graduation criteria.

Reply:
Added to non-goals.